Source/file.swift

```diff
@@ -0,0 +1,5 @@
+class scShortVariableName {
```

Comment:
Type Name Violation: Type name should start with an uppercase character: 'scShortVariableName' (type_name)